src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.trim().split(';');
+
+  const stylesObject = {};
+
+  stylesArray.forEach((style) => {
+    if (style.trim() === '') {
```

Comment:
```suggestion
    if (!style.trim()) {
```
You don't need to compare with an empty string, just check if value exists